src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/InstantiateArgumentExceptionsCorrectlyTests.cs

```diff
@@ -106,17 +106,24 @@ End Sub
         }
 
         [Fact]
-        public async Task ArgumentException_ParameterNameAsMessage_Warns()
+        public async Task ArgumentException_ParameterNameAsMessage_WarnsAndCodeFixesWithNameOf()
         {
-            await VerifyCS.VerifyAnalyzerAsync(@"
+            await VerifyCS.VerifyCodeFixAsync(@"
                 public class Class
                 {
                     public void Test(string first)
                     {
                         throw new System.ArgumentException(""first"");
                     }
                 }",
-                GetCSharpIncorrectMessageExpectedResult(6, 31, "Test", "first", "message", "ArgumentException"));
+                GetCSharpIncorrectMessageExpectedResult(6, 31, "Test", "first", "message", "ArgumentException"), @"
+                public class Class
+                {
+                    public void Test(string first)
+                    {
+                        throw new System.ArgumentException(null, nameof(first));
```

Comment:
Should we generate a named argument for 'null', say `message: null`? Alternatively, should we use a dummy message string to help inform users that they should likely provide an argument message, say `"message-to-be-filled" or some such (with better wording)?

Reply:
`null` is suggested by in the issue CC @terrajobst @bartonjs

Reply:
> should we use a dummy message string?

`null` results in using the default ArgumentException message, so it's actually quite likely what the caller wants.

> Should we generate a named argument for 'null', say message: null? 

I wouldn't.  (Unless there's an editorconfig style value for this, that the analyzer can see... in which case we should match it).